hr_employee_birth_name/models/hr_employee.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# copyright  2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    hr_employee_birth_name = fields.Char(
```

Comment:
Why not calling this field simply `birth_name` ?

Like in https://github.com/OCA/partner-contact/tree/7.0/partner_birth_name